### ISSUE TITLE
As a funding report user, I can view an overview page of monthly funding reports

### ISSUE THREAD

**Issue by Derekskylight** [ctoec/data-collection#1433]
## Background 
We are finally at the point where we can go on the 'offensive' towards funding reports and are able to start upon the initial stages of its development!  

Here lies the initial stage of the reporting user journey in which the funding report user will be able to verify that their roster is complete for that month, which will then progress them towards the next step of their journey to review the funding report and eventually confirm it 

## Acceptance Criteria
- Title including _month_ and text below for the _funding stream and individual site name_ 
- Tabulated data including _Report Month, Funding Source, Site, and Status_
- Progress bar indicating what stage the user is on. At this stage, only the bar including _confirm roster_ will be filled out. 
- Header text and instructional text below describing text for the user. 
- Tabulated data including columns such as _Name, Age group, Birthdate, Start enrollment date, and Attendance_
- Buttons including _Back and Next_ to move backward or forwards in the journey. 

## Images
Link to [Figma File here](https://www.figma.com/file/bryUsb6TQnrc6Xiyj3wnEX/ECE-Reporter-%E2%80%93-WIP-Designs?node-id=809%3A8813)
![Monthly Reporting 2.PNG](https://images.zenhubusercontent.com/602d78337403e44d7db5c4f3/98ebf3ef-a4bd-43ba-903b-f29fd09b4386)

**Comment by JordanGuinn** [comment]
@Derekskylight @ehuang525 We decided that we wanted all of the `Attended at least one day this month` checkboxes for all children to be checked **by default**, correct?

**Comment by ehuang525** [comment]
@JordanGuinn yes! Providers can then uncheck for the children that don't satisfy that criteria since it seems more common that they _would_ satisfy that criteria.
